packages/components/app/styles/components/dropdown.scss

```diff
@@ -18,27 +14,24 @@
 // DROPDOWN COMPONENT
 //
 // properties within each class are sorted alphabetically
-// notice: pseudo-classes for the states *must* follow the order link > visited > focus > hover > active
+// notice: pseudo-classes for the states *must* follow the order link > visited > hover > focus > active
```

Comment:
[chore]: in all the other components we're saying something different. if we change it here, let's change it across all the other components

Reply:
"lord vader hates fluffy animals" is the mnemonic device I use, it's possible I mixed it up before, noticed it today.